client/src/components/UpgradeNotification/index.js

```diff
@@ -0,0 +1,40 @@
+import { versionOutOfDate } from '../../utils/version';
+
+const initUpgradeNotification = () => {
+  const container = document.querySelector('[data-upgrade]');
+
+  if (!container) {
+    return;
+  }
+
+  /*
+  *   Expected JSON payload:
+  *   {
+  *       "version" : "1.2.3",    // Version number. Can only contain numbers and decimal point.
+  *       "url" : "https://wagtail.io"  // Absolute URL to page/file containing release notes or actual package. It's up to you.
+  *   }
+  */
+  const releasesUrl = 'https://releases.wagtail.io/latest.txt';
```

Comment:
Just a thought - would there be any benefit in storing the releaseUrl on the container also?

benefits - can be customisable by devs without needing to load different JS (edge case probably), means the function itself contains less data hard coded (eventually the URL could be pulled out further into a tag or constant elsewhere.

Reply:
Yeah sure. I think this is a good idea - off the top of my head I can see how this might be useful for CodeRed style projects.